crates/rip_bin/src/main.rs

```diff
@@ -40,8 +43,17 @@ struct Args {
 
     #[clap(flatten)]
     sdist_resolution: SDistResolution,
+    
+    #[arg(short = 'e', long, value_parser = parse_key_val::<String>)]
+    /// Enviroment variables that should be used when building wheels.
+    /// Can be passed using KEY=VALUE
+    enviroment_variables: HashMap<String, String>,
```

Comment:
Like I said in `rip_bin` itself I would just rather use `MY_VAR=bla cargo r my_package` e.g.